PrivateClinicsNetWebApi/Controllers/RoleManagmentConroller.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using PrivateClinicsWebNet.Application.DTOs;
+using PrivateClinicsWebNet.Application.Services;
+
+namespace PrivateClinicsNetWebApi.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class RoleManagmentConroller : ControllerBase
+    {
+        private readonly RoleService _roleService;
+
+        public RoleManagmentConroller(RoleService roleService)
+        {
+            _roleService = roleService;
+        }
+
+        [HttpPost("role-create")]
+        public async Task<IActionResult> CreateRoleAsync([FromBody] RoleCreateDto model)
+        {
+            try
+            {
+                await _roleService.CreateRoleAsync(model.RoleName);
+                return Ok("Role created successfully");
+            }
+            catch (Exception ex)
+            {
+                return BadRequest(ex.Message);
+            }
+        }
```

Comment:
We don't want to create roles dynamically but to have roles specified in domain, like class with constant strings. Then u can either seed them on startup or add them to migration using builder.Entity<Role>.HasData(<roles)